.circleci/config.yml

```diff
@@ -85,7 +85,9 @@ jobs:
       - *attach_workspace
       - run:
           name: Lint
-          command: yarn lint
+          command: |
+            yarn lint
+            yarn eslint
```

Comment:
Could we do these as parallel circle CI tasks?
(just like how the tests are as separate circle ci tasks and therefore get run in parallel)

Reply:
Yep, all changes were made by eslint. TSLint also has [this rule](https://palantir.github.io/tslint/rules/interface-over-type-literal/)

Reply:
> Could we do these as parallel circle CI tasks?
> (just like how the tests are as separate circle ci tasks and therefore get run in parallel)

Sure

Reply:
Thanks!